compiler-rt/lib/builtins/int_lib.h

```diff
@@ -49,7 +49,7 @@
 #define SYMBOL_NAME(name) XSTR(__USER_LABEL_PREFIX__) #name
 
 #if defined(__ELF__) || defined(__MINGW32__) || defined(__wasm__) ||           \
-    defined(_AIX)    || defined(__CYGWIN__)
+    defined(_AIX) || defined(__CYGWIN__) || defined(__UEFI__)
```

Comment:
I'm not sure that this is the right branch to target UEFI. UEFI doesn't use ELF and doesn't guarantee the GNU syntax. What if you completely disable aliasing as per the Windows path on L67?

Reply:
Yeah, I'm not sure either where's a good spot to put it. Probably is best on the safe side to put it along with the Windows one.